src/pages/latestprojects/Components/NoteCard.tsx

```diff
@@ -39,12 +39,18 @@ export default function NoteCard({ nostrich, content, date }: Props) {
       <Flex
         className="notecard"
         alignItems="flex-start"
+        overflow="hidden"
         margin="16px 0"
         paddingTop="0px"
-        border="1px solid #ccc"
+        border="2px solid #ccc"

```

Comment:
dont use actual colors, use borderColor: neutral.200. Also, it might be a good idea to use the CardLayout component, which is already as per standard, that'll make things easier.